src/Components/CustomHooks/PushBackOnRefresh.ts

```diff
@@ -10,5 +10,5 @@ export const usePushBackOnRefresh = () => {
       navigate('/create-or-manage-page');
```

Comment:
If you did need a useEffect like this I'd avoid tucking it away in a custom hook like this. I'm sure it depends on the circumstances but it just further obscures the effect and makes it a one off call that the reader has to click through to see whats actually happening. if it was a massive callback in the useEffect you could break that out into a separate function below the component or in another file

Reply:
I remember it feeling kinda wrong when I wrote it.
Im pretty sure I had just watched a tutorial on custom hooks and was looking for any excuse to make one, lol.

Good point for me tp remember, that if seems like a custom hook needs an effect inside it I should be looking for another way to achieve whatever im doing.